src/keeshare/SettingsWidgetKeeShare.ui

```diff
@@ -215,27 +252,39 @@
         </item>
         <item>
          <widget class="QPushButton" name="trustImportedCertificateButton">
+          <property name="accessibleName">
+           <string>Trust selected certificate</string>
+          </property>
           <property name="text">
            <string>Trust</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="askImportedCertificateButton">
+          <property name="accessibleName">
+           <string>Ask trust for selected certificate</string>
```

Comment:
Maybe something along the lines of "Ask whether to trust certificate every time"?